python/foglamp/plugins/storage/postgres/backup_restore/backup_postgres.py

```diff
@@ -35,6 +37,13 @@
 
 _MODULE_NAME = "foglamp_backup_postgres_module"
 
+_LOGGER = logger.setup(_MODULE_NAME, level=20)
+
+# to avoid multiple messages
+_LOGGER.handlers = []
+_LOGGER = logger.setup(_MODULE_NAME, level=20)
```

Comment:
I observed multiples identical lines logged and I wasn't able to identify the cause, I found that doing a reinitialization solved the problem

Reply:
I dug in the problem, it is no more there so I removed the rows 

# to avoid multiple messages
_LOGGER.handlers = []
_LOGGER = logger.setup(_MODULE_NAME, level=20)

Reply:
solved